servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -97,17 +114,18 @@ def _to_top_labels_aggregations(label_summaries):
     # where valid means a label has jobs more than _LABEL_MIN_COUNT_FOR_RANK.
     label_freq = {}
     for label, item in label_summaries.items():
+        # Do not use the job-id or task-id label as aggregation key
```

Comment:
nit: in general comments should be used to explain "why" rather than "what"; this current comment basically just restates the code. Something like "job-id and task-id are assigned by dsub and are not meaningful for aggregation"